### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by amodesKontist**

### CODE REVIEW

File: lib/graphql/schema.flow.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* @flow */
+// @flow
```
Comment (Campalo) [inline lib/graphql/schema.flow.js:1]: Why changing the comment syntax ? Both are fine, just wondering :)

**Review by Campalo — APPROVED** [review]
👍

File: lib/graphql/schema.flow.js [hunk 2]
```diff
@@ -30,7 +30,6 @@ export type User = {|
   __typename?: 'User',
   email: $ElementType<Scalars, 'String'>,
   createdAt: $ElementType<Scalars, 'DateTime'>,
-  vatCutoffLine?: ?$ElementType<Scalars, 'DateTime'>,
```
Comment (Campalo) [inline lib/graphql/schema.flow.js:33]: I saw that this deleted line is linked to your previous prs from BE and webapp 👍

File: lib/graphql/schema.flow.js (hunk 1 above)
Comment (amodesKontist) [reply lib/graphql/schema.flow.js:1]: I just ran `yarn schema:generate`  🤔

File: lib/graphql/schema.flow.js (hunk 2 above)
Comment (amodesKontist) [reply lib/graphql/schema.flow.js:33]: yes its the reason why I do this update.
Problem is that there is a test on BE that uses the sdk to request `vatCutoffline`.
So the test from this PR always fails https://github.com/kontist/backend/pull/9220 so I update the sdk first. 
Kind of circular depenedency between the test and sdk 😅

**Pull request merged**